l10n_it_fatturapa_out_di/tests/test_fatturapa_xml_validation.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2021 Marco Colombo https://github.com/TheMule71
+
+import base64
+
+from odoo import fields
+from odoo.tests import Form
+
+from .fatturapa_common import FatturaPACommon
+
+
+class TestFatturaPAXMLValidation(FatturaPACommon):
+    def setUp(self):
+        super(TestFatturaPAXMLValidation, self).setUp()
+
+        # XXX - a company named "YourCompany" alread exists
```

Comment:
Non si può dare un nome diverso direttamente alla nuova company? Mi sfugge la logica.

Reply:
Ne abbiamo discusso, la cosa va sistemata in #2507.

La nuova company ha un nome diverso, ma tutti gli XML di test (dalla 12) hanno YourCompany all'interno.  Invece di modificare tutti gli XML dei test, ho preferito rinominare la company usata (che ha tutti i dati necessari per emettere fatture elettroniche) in "YourCompany".

In questo modulo semplicemente la cosa è stata ereditata, avrei potuto usare un nome diverso nell'XML, ma per coerenza tengo l'XML simile a quello degli altri test di fatturapa_out.

Reply:
Perfetto. Grazie per le info di cui non ero a conoscenza.